### ISSUE TITLE
Command execution error got ignored - "Exec command ... is already running"

### ISSUE THREAD

**Issue by rsotnychenko** [nektos/act#165]
`act` failed to execute a step and reported it as successful:
```
[Build: project-1/project-1-application    ] ⭐  Run INIT: compute DOCKER_IMAGE_NAME var
[Build: project-1/project-1-application    ] Extracting content to '/github/'
[Build: project-1/project-1-application    ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/2]'
| Error: Exec command 08310b325ae814b236e1ef9e4b54663af06e49e70a9f18abf4900d88bba17c63 is already running
[Build: project-1/project-1-application    ]   ✅  Success - INIT: compute DOCKER_IMAGE_NAME var
```

This behaviour happened to me only once (job succeed after a restart), so I suspect some kind of a race condition going on here. Looks like `Error: Exec command failed ...` was logged here https://github.com/nektos/act/blob/bf3824cc106a6f02504e20f673f2f370c4450679/pkg/container/docker_run.go#L294-L296

Workflow:
```
# NOTE: This file is autogenerated. Do NOT edit it manually!
name: 'Build: project-1'
'on':
  push: {}
env:
  DOCKER_REGISTRY_HOST_DEV: docker.pkg.github.com
  DOCKER_IMAGE_PATH: ${{ github.repository }}
  DEFAULT_BRANCH: develop
  MODULE_PATH: .
jobs:
  project-1-base:
    runs-on: ubuntu-18.04
    env:
      JOB_NAME: project-1-base
      SUBMODULE_NAME: base
    steps:
    - name: 'INIT: compute LOCAL_RUN var'
      run: if [ "$GITHUB_ACTOR" = "nektos/act" ]; then X=yes; else X=no; fi; echo
        "::set-env name=LOCAL_RUN::$X"
    - name: 'INIT: compute CURRENT_BRANCH var'
      run: echo "::set-env name=CURRENT_BRANCH::$(echo $GITHUB_REF | cut -d '/' -f3-)"
    - name: 'INIT: compute DOCKER_IMAGE_NAME var'
      run: echo "::set-env name=DOCKER_IMAGE_NAME::$DOCKER_IMAGE_PATH/$JOB_NAME"
    - name: 'INIT: compute DOCKER_IMAGE_TAG var'
      run: if [ "$LOCAL_RUN" = "yes" ]; then X="latest"; else X="$GITHUB_SHA"; fi;
        echo "::set-env name=DOCKER_IMAGE_TAG::$X"
    - uses: actions/checkout@v2
    ... redacted ...
    timeout-minutes: 10
  project-1-application:
    runs-on: ubuntu-18.04
    env:
      JOB_NAME: project-1-application
      SUBMODULE_NAME: application
    steps:
    - name: 'INIT: compute LOCAL_RUN var'
      run: if [ "$GITHUB_ACTOR" = "nektos/act" ]; then X=yes; else X=no; fi; echo
        "::set-env name=LOCAL_RUN::$X"
    - name: 'INIT: compute CURRENT_BRANCH var'
      run: echo "::set-env name=CURRENT_BRANCH::$(echo $GITHUB_REF | cut -d '/' -f3-)"
    - name: 'INIT: compute DOCKER_IMAGE_NAME var'
      run: echo "::set-env name=DOCKER_IMAGE_NAME::$DOCKER_IMAGE_PATH/$JOB_NAME"
    - name: 'INIT: compute DOCKER_IMAGE_TAG var'
      run: if [ "$LOCAL_RUN" = "yes" ]; then X="latest"; else X="$GITHUB_SHA"; fi;
        echo "::set-env name=DOCKER_IMAGE_TAG::$X"
    - uses: actions/checkout@v2
    ... redacted ...
    timeout-minutes: 10
  project-1-cron:
    runs-on: ubuntu-18.04
    env:
      JOB_NAME: project-1-cron
      SUBMODULE_NAME: cron
    steps:
    ... redacted ...
    timeout-minutes: 10
    needs:
    - project-1-base
    - project-1-application
  project-1-msa:
    runs-on: ubuntu-18.04
    env:
      JOB_NAME: project-1-msa
      SUBMODULE_NAME: msa
    steps:
    ... redacted ...
    timeout-minutes: 10
    needs:
    - project-1-base
    - project-1-application
  project-1-file-processing:
    runs-on: ubuntu-18.04
    env:
      JOB_NAME: project-1-file-processing
      SUBMODULE_NAME: file-processing
    steps:
    ... redacted ...
    timeout-minutes: 10
    needs:
    - project-1-base
    - project-1-application
  project-1-ws:
    runs-on: ubuntu-18.04
    env:
      JOB_NAME: project-1-ws
      SUBMODULE_NAME: ws
    steps:
    ... redacted ...
    timeout-minutes: 10
    needs:
    - project-1-base
    - project-1-application
```

Full log:
```
$> act -v -s GITHUB_TOKEN=REDACTED -P ubuntu-18.04=nektos/act-environments-ubuntu:18.04
DEBU[0000] Loading workflows from '/Users/USERNAME-REDACTED/Projects/platform/project-1/.github/workflows'
DEBU[0000] Reading workflow '/Users/USERNAME-REDACTED/Projects/platform/project-1/.github/workflows/build-project-1.yml'
DEBU[0000] Planning event: push
DEBU[0000] Looking for a git repository in: /Users/USERNAME-REDACTED/Projects/platform/project-1
DEBU[0000] Found revision: d4b69070a1edf7e45bc80613f73105aa1df69f71
DEBU[0000] using github ref: refs/heads/develop
... REDACTED: Looking for a git repository & found revision & using github ref (x4) ...
[Build: project-1/project-1-application    ] 🚀  Start image=nektos/act-environments-ubuntu:18.04
[Build: project-1/project-1-base           ] 🚀  Start image=nektos/act-environments-ubuntu:18.04
... REDACTED: Looking for a git repository & found revision & using github ref (x2) ...
[Build: project-1/project-1-base           ]   🐳  docker pull nektos/act-environments-ubuntu:18.04
[Build: project-1/project-1-application    ]   🐳  docker pull nektos/act-environments-ubuntu:18.04
DEBU[0000] Image exists? true
DEBU[0000] Image exists? true
[Build: project-1/project-1-application    ]   🐳  docker create image=nektos/act-environments-ubuntu:18.04 entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Build: project-1/project-1-base           ]   🐳  docker create image=nektos/act-environments-ubuntu:18.04 entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Build: project-1/project-1-application    ] Created container name=act-Build--project-1-project-1-a id=97f4ec1611700e4ebcf928d7758b22e89d64dbddb70e29b8bf64b508a1e62ba9 from image nektos/act-environments-ubuntu:18.04
[Build: project-1/project-1-application    ] ENV ==> [RUNNER_TOOL_CACHE=/opt/hostedtoolcache]
[Build: project-1/project-1-application    ]   🐳  docker run image=nektos/act-environments-ubuntu:18.04 entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Build: project-1/project-1-application    ] Starting container: 97f4ec1611700e4ebcf928d7758b22e89d64dbddb70e29b8bf64b508a1e62ba9
[Build: project-1/project-1-base           ] Created container name=act-Build--project-1-project-1-b id=cb521c0901f6c794421f7ac07092fa8ad0f0837164d40d7be0da97a988123859 from image nektos/act-environments-ubuntu:18.04
[Build: project-1/project-1-base           ] ENV ==> [RUNNER_TOOL_CACHE=/opt/hostedtoolcache]
[Build: project-1/project-1-base           ]   🐳  docker run image=nektos/act-environments-ubuntu:18.04 entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Build: project-1/project-1-base           ] Starting container: cb521c0901f6c794421f7ac07092fa8ad0f0837164d40d7be0da97a988123859
[Build: project-1/project-1-base           ] Started container: cb521c0901f6c794421f7ac07092fa8ad0f0837164d40d7be0da97a988123859
[Build: project-1/project-1-base           ]   🐳  docker cp src=/Users/USERNAME-REDACTED/Projects/platform/project-1/. dst=/github/workspace
[Build: project-1/project-1-base           ] Exec command '[mkdir -p /github/workspace]'
[Build: project-1/project-1-application    ] Started container: 97f4ec1611700e4ebcf928d7758b22e89d64dbddb70e29b8bf64b508a1e62ba9
[Build: project-1/project-1-application    ]   🐳  docker cp src=/Users/USERNAME-REDACTED/Projects/platform/project-1/. dst=/github/workspace
[Build: project-1/project-1-application    ] Exec command '[mkdir -p /github/workspace]'
DEBU[0000] Writing tarball /var/folders/hf/t7sdflz10fs7mhm62z1lbr480000gn/T/act296766640 from /Users/USERNAME-REDACTED/Projects/platform/project-1/.
DEBU[0000] Stripping prefix:/Users/USERNAME-REDACTED/Projects/platform/project-1/ src:/Users/USERNAME-REDACTED/Projects/platform/project-1/.
DEBU[0000] Writing tarball /var/folders/hf/t7sdflz10fs7mhm62z1lbr480000gn/T/act266235983 from /Users/USERNAME-REDACTED/Projects/platform/project-1/.
DEBU[0000] Stripping prefix:/Users/USERNAME-REDACTED/Projects/platform/project-1/ src:/Users/USERNAME-REDACTED/Projects/platform/project-1/.
[Build: project-1/project-1-application    ] Extracting content from '/var/folders/hf/t7sdflz10fs7mhm62z1lbr480000gn/T/act266235983' to '/github/workspace'
[Build: project-1/project-1-base           ] Extracting content from '/var/folders/hf/t7sdflz10fs7mhm62z1lbr480000gn/T/act296766640' to '/github/workspace'
DEBU[0007] Writing entry to tarball workflow/event.json len:2
DEBU[0007] Writing entry to tarball home/.act len:0
[Build: project-1/project-1-application    ] Extracting content to '/github/'
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0007] setupEnv => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application MODULE_PATH:. SUBMODULE_NAME:application]
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0007] context env => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application MODULE_PATH:. SUBMODULE_NAME:application]
[Build: project-1/project-1-application    ] ⭐  Run INIT: compute LOCAL_RUN var
DEBU[0007] Wrote command 'if [ "$GITHUB_ACTOR" = "nektos/act" ]; then X=yes; else X=no; fi; echo "::set-env name=LOCAL_RUN::$X"' to 'workflow/0'
DEBU[0007] Writing entry to tarball workflow/0 len:117
[Build: project-1/project-1-application    ] Extracting content to '/github/'
DEBU[0007] Writing entry to tarball workflow/event.json len:2
DEBU[0007] Writing entry to tarball home/.act len:0
[Build: project-1/project-1-base           ] Extracting content to '/github/'
[Build: project-1/project-1-application    ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/0]'
[Build: project-1/project-1-application    ]   ⚙  ::set-env:: LOCAL_RUN=yes
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0007] setupEnv => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-base MODULE_PATH:. SUBMODULE_NAME:base]
... REDACTED: Looking for a git repository & found revision & using github ref ...
[Build: project-1/project-1-application    ]   ✅  Success - INIT: compute LOCAL_RUN var
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0007] context env => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-base MODULE_PATH:. SUBMODULE_NAME:base]
[Build: project-1/project-1-base           ] ⭐  Run INIT: compute LOCAL_RUN var
DEBU[0007] Wrote command 'if [ "$GITHUB_ACTOR" = "nektos/act" ]; then X=yes; else X=no; fi; echo "::set-env name=LOCAL_RUN::$X"' to 'workflow/0'
DEBU[0007] Writing entry to tarball workflow/0 len:117
[Build: project-1/project-1-base           ] Extracting content to '/github/'
DEBU[0007] using github ref: refs/heads/develop
DEBU[0007] setupEnv => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:application]
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0007] context env => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:application]
[Build: project-1/project-1-application    ] ⭐  Run INIT: compute CURRENT_BRANCH var
DEBU[0007] Wrote command 'echo "::set-env name=CURRENT_BRANCH::$(echo $GITHUB_REF | cut -d '/' -f3-)"' to 'workflow/1'
DEBU[0007] Writing entry to tarball workflow/1 len:91
[Build: project-1/project-1-application    ] Extracting content to '/github/'
[Build: project-1/project-1-base           ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/0]'
[Build: project-1/project-1-application    ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/1]'
[Build: project-1/project-1-base           ]   ⚙  ::set-env:: LOCAL_RUN=yes
[Build: project-1/project-1-application    ]   ⚙  ::set-env:: CURRENT_BRANCH=develop
[Build: project-1/project-1-base           ]   ✅  Success - INIT: compute LOCAL_RUN var
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0008] setupEnv => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-base LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:base]
... REDACTED: Looking for a git repository & found revision & using github ref ...
[Build: project-1/project-1-application    ]   ✅  Success - INIT: compute CURRENT_BRANCH var
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0008] context env => map[DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-base LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:base]
[Build: project-1/project-1-base           ] ⭐  Run INIT: compute CURRENT_BRANCH var
DEBU[0008] Wrote command 'echo "::set-env name=CURRENT_BRANCH::$(echo $GITHUB_REF | cut -d '/' -f3-)"' to 'workflow/1'
DEBU[0008] Writing entry to tarball workflow/1 len:91
[Build: project-1/project-1-base           ] Extracting content to '/github/'
DEBU[0008] using github ref: refs/heads/develop
DEBU[0008] setupEnv => map[CURRENT_BRANCH:develop DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:2 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:application]
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0008] context env => map[CURRENT_BRANCH:develop DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:2 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:application]
[Build: project-1/project-1-application    ] ⭐  Run INIT: compute DOCKER_IMAGE_NAME var
DEBU[0008] Wrote command 'echo "::set-env name=DOCKER_IMAGE_NAME::$DOCKER_IMAGE_PATH/$JOB_NAME"' to 'workflow/2'
DEBU[0008] Writing entry to tarball workflow/2 len:85
[Build: project-1/project-1-application    ] Extracting content to '/github/'
[Build: project-1/project-1-base           ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/1]'
[Build: project-1/project-1-application    ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/2]'
[Build: project-1/project-1-base           ]   ⚙  ::set-env:: CURRENT_BRANCH=develop
| Error: Exec command 08310b325ae814b236e1ef9e4b54663af06e49e70a9f18abf4900d88bba17c63 is already running
[Build: project-1/project-1-application    ]   ✅  Success - INIT: compute DOCKER_IMAGE_NAME var
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0008] setupEnv => map[CURRENT_BRANCH:develop DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:3 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:application]
... REDACTED: Looking for a git repository & found revision & using github ref ...
[Build: project-1/project-1-base           ]   ✅  Success - INIT: compute CURRENT_BRANCH var
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0008] context env => map[CURRENT_BRANCH:develop DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:3 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-application LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:application]
[Build: project-1/project-1-application    ] ⭐  Run INIT: compute DOCKER_IMAGE_TAG var
DEBU[0008] Wrote command 'if [ "$LOCAL_RUN" = "yes" ]; then X="latest"; else X="$GITHUB_SHA"; fi; echo "::set-env name=DOCKER_IMAGE_TAG::$X"' to 'workflow/3'
DEBU[0008] Writing entry to tarball workflow/3 len:130
[Build: project-1/project-1-application    ] Extracting content to '/github/'
DEBU[0008] using github ref: refs/heads/develop
DEBU[0008] setupEnv => map[CURRENT_BRANCH:develop DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:2 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-base LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:base]
... REDACTED: Looking for a git repository & found revision & using github ref ...
DEBU[0008] context env => map[CURRENT_BRANCH:develop DEFAULT_BRANCH:develop DOCKER_IMAGE_PATH:organization/project-1 DOCKER_REGISTRY_HOST_DEV:docker.pkg.github.com GITHUB_ACTION:2 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:refs/heads/develop GITHUB_REPOSITORY:organization/project-1 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:d4b69070a1edf7e45bc80613f73105aa1df69f71 GITHUB_TOKEN:TOKENREDACTED GITHUB_WORKFLOW:Build: project-1 GITHUB_WORKSPACE:/github/workspace HOME:/github/home JOB_NAME:project-1-base LOCAL_RUN:yes MODULE_PATH:. SUBMODULE_NAME:base]
[Build: project-1/project-1-base           ] ⭐  Run INIT: compute DOCKER_IMAGE_NAME var
DEBU[0008] Wrote command 'echo "::set-env name=DOCKER_IMAGE_NAME::$DOCKER_IMAGE_PATH/$JOB_NAME"' to 'workflow/2'
DEBU[0008] Writing entry to tarball workflow/2 len:85
[Build: project-1/project-1-base           ] Extracting content to '/github/'
[Build: project-1/project-1-application    ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/3]'
[Build: project-1/project-1-base           ] Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/2]'
[Build: project-1/project-1-application    ]   ⚙  ::set-env:: DOCKER_IMAGE_TAG=latest
[Build: project-1/project-1-base           ]   ⚙  ::set-env:: DOCKER_IMAGE_NAME=organization/project-1/project-1-base
[Build: project-1/project-1-application    ]   ✅  Success - INIT: compute DOCKER_IMAGE_TAG var
... REDACTED: Looking for a git repository & found revision & using github ref ...
[Build: project-1/project-1-base           ]   ✅  Success - INIT: compute DOCKER_IMAGE_NAME var
...
build continued and failed later on due to missing DOCKER_IMAGE_NAME variable
...
```

**Comment by cplee** [comment]
I think the problem here is the generated container names are conflicting with each other

**Comment by cplee** [comment]
should be fixed by #174

**Comment by spl** [comment]
I'm see something like this, too:

```
Exec command '[bash --noprofile --norc -eo pipefail /github/workflow/0]'
  | Error: Exec command ... is already running
```

It happens only on the 7th or so instantiation of the same image.

I'm using `act` version 0.2.16 with [Actionsflow](https://github.com/actionsflow/actionsflow).
